can-rest-model.md

```diff
@@ -9,30 +9,26 @@ Connect a type to a restful service layer.
 
 `restModel` extends the provided `options.Map` type
 with the ability to connect to a restful service layer. For example,
-the following defines a `Todo` and `TodoList` type and extends them
+the following extends a `Todo` type
```

Comment:
Generally speaking, we have used `define` and `instantiate`.  Thoughts @chasenlehara / @phillipskevin / @matthewp ?  As we move to `class` we could say "extend".  Or we could start now?

Reply:
extend seems accurate to me regardless of how the inheritance happens.